crates/python/qcs_sdk/_execution_data.pyi

```diff
@@ -0,0 +1,39 @@
+"""
+Do not import this file, it has no exports.
+It is only here to represent the structure of the rust source code 1:1
+"""
+
+import datetime
+from typing import List, Optional
+
+class IntegerReadoutValues:
+    values: List[int]
+
+
+class ComplexReadoutValues:
+    values: List[complex]
+
+
+class ReadoutValuesValues:
+    integer_values: IntegerReadoutValues
+    complex_values: ComplexReadoutValues
```

Comment:
I'm not sure these are actually being exported to Python in the Rust source. Need to go through the types that _are_ exported and make sure all the types they reference are _also_ exported.

Reply:
That's what I was trying to get at with the `Do not import this file...` comment at the top - you aren't able to directly import these structs, but you will encounter these types from `QPU.readout_data.get_readout_values(field, index)`.

Since https://typing.readthedocs.io/en/latest/source/stubs.html#private-definitions suggest that these can be implied to be private by prefixing with an underscore, so I can do that.